src/govuk/components/notification-banner/template.njk

```diff
@@ -0,0 +1,40 @@
+{% if params.role %}
+  {% set role = params.role %}
+{% elif params.type == "success" or params.type == "error" %}
+  {% set role = "alert" %}
+{% else %}
+  {% set role = "region" %}
+{% endif %}
+
+{% if params.tabindex %}
+  {% set tabindex = params.tabindex %}
+{% elif params.type == "success" or params.type == "error" %}
+  {% set tabindex = "-1" %}
+{% endif %}
+
+{% set titleHeadingLevel = params.titleHeadingLevel if params.titleHeadingLevel else 2 %}
+
+<div class="govuk-notification-banner
+  {%- if params.type == "success" %} govuk-notification-banner--success{% endif %}
+  {%- if params.type == "error" %} govuk-notification-banner--error{% endif %}
+  {%- if params.classes %} {{ params.classes }}{% endif %}"
+  role="{{ role }}"
+  {% if tabindex %} tabindex="{{ tabindex }}" {% endif %}
+  {% if not(params.type == "success") and not(params.type == "error") %} aria-labelledby="govuk-notification-banner-title" {% endif %}
+  {% if ((params.type == "success" or params.type == "error") and not(params.initialFocus === false)) or params.initialFocus === true %} data-module="govuk-initial-focus"{% endif %}
+  {% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}
+>
+  <div class="govuk-notification-banner__header">
+    <h{{ titleHeadingLevel }} class="govuk-notification-banner__title" {% if not(params.type == "success") and not(params.type == "error") %} id="govuk-notification-banner-title" {% endif %}
+    >
+      {%- if params.titleHtml %} {{ params.titleHtml | safe }}
+      {%- elif params.title %} {{ params.title }}
+      {%- elif params.type == "success" %}Success
+      {%- elif params.type == "error" %}Error
+      {%- else %}Important{%- endif -%}
+    </h{{ titleHeadingLevel }}>
+  </div>
+  <div class="govuk-notification-banner__content">
+    {{ params.contentHtml | safe if params.contentHtml else params.content }}
```

Comment:
Should we render`content.text`inside a `<p>` tag? I've just been looking at the styling, and having `content.text` inside it's own paragraph would enable us to style it without then automatically applying those styles to any `content.html` a user passes in (and therefore forcing them to override our styling)

Reply:
@vanitabarrett I think wrapping the default text in `<p>` tags would make sense semantically. From a styling point of view though what if the user passes `<p>` tags as part of the `html` option? Would it be unexpected that those get styled in a certain way? Let me know what you think as you have a better idea of how the styles are going to work.

Reply:
@hannalaakso I was thinking if we wrap `contentText` in a `<p>` tag that gets a specific class, e.g: `govuk-notification-banner__text`, then we can target that class specifically and leave any `contentHtml` unstyled. That would make it the responsibility of the user to style any html that they pass into the component.

Reply:
Thanks @vanitabarrett. Following our chat offline I've wrapped the `<p>` tag in a class. That class can then be used for styling the `<p>` tags in html that need to look like headings.